src/main/java/com/github/msemys/esjc/EventStoreBuilder.java

```diff
@@ -213,6 +213,18 @@ public EventStoreBuilder useSslConnection(String certificateCommonName) {
         return this;
     }
 
+    /**
+     * Enables connection encryption using SSL and trusts an X.509 server certificate whose certificate is trusted
+     * by the given certificate file (in PEM form).
+     *
+     * @param certificateFile server certificate
+     * @return the builder reference
+     */
+    public EventStoreBuilder useSslConnectionCertificateFile(String certificateFile) {
```

Comment:
suggesting api change
```suggestion
    public EventStoreBuilder useSslConnection(File certificateFile)
```